contracts/EscrowAndStaking/StakingThales.sol

```diff
@@ -124,151 +136,183 @@ contract StakingThales is IStakingThales, Initializable, ProxyOwned, ProxyReentr
 
     /* ========== VIEWS ========== */
 
+    /// @notice Get the total staked amount on the contract
+    /// @return total staked amount
     function totalStakedAmount() external view returns (uint) {
         return _totalStakedAmount;
     }
 
+    /// @notice Get the staked balance for the account
+    /// @param account to get the staked balance for
+    /// @return the staked balance for the account
     function stakedBalanceOf(address account) external view returns (uint) {
         return _stakedBalances[account];
     }
 
+    /// @notice Get the last period of claimed rewards for the account
+    /// @param account to get the last period of claimed rewards for
+    /// @return the last period of claimed rewards for the account
     function getLastPeriodOfClaimedRewards(address account) external view returns (uint) {
         return _lastRewardsClaimedPeriod[account];
     }
 
-    function getRewardsAvailable(address account) external view returns (uint) {
+    /// @notice Get the rewards available for the claim for the account
+    /// @param account to get the rewards available for the claim for
+    /// @return the rewards available for the claim for the account
+    function getRewardsAvailable(address account) public view returns (uint) {
```

Comment:
Better to leave it external and use the internal function in line 741.
Mainly because public functions use double the gas and copy arguments to memory.

Reply:
Yes, I missed internal function somehow. Updated.